services/minespace-web/src/components/dashboard/mine/overview/Overview.js

```diff
@@ -42,145 +44,206 @@ export class Overview extends Component {
-              </Text>
-            </Paragraph>
-            <Paragraph>
-              <Text strong className="ministry-contact-title">
-                Senior Permitting Inspector
-              </Text>
-              <br />
-              <Text>{this.state.contacts.permitting.name}</Text>
-              <br />
-              <Text>{this.state.contacts.permitting.phone}</Text>
-              <br />
-              <Text>
-                <a href={`mailto:${this.state.contacts.permitting.email}`}>
-                  {this.state.contacts.permitting.email}
-                </a>
-              </Text>
-            </Paragraph>
-            <Paragraph>
-              <Text strong className="ministry-contact-title">
-                Regional Director
-              </Text>
-              <br />
-              <Text>{this.state.contacts.director.name}</Text>
-              <br />
-              <Text>{this.state.contacts.director.phone}</Text>
-              <br />
-              <Text>
-                <a href={`mailto:${this.state.contacts.director.email}`}>
-                  {this.state.contacts.director.email}
-                </a>
-              </Text>
-            </Paragraph>
-            <Paragraph>
-              <Text strong className="ministry-contact-title">
-                Regional Office
-              </Text>
-              <br />
-              <Text>{this.state.contacts.office.phone}</Text>
-              <br />
-              <Text>
-                <a href={`mailto:${this.state.contacts.office.email}`}>
-                  {this.state.contacts.office.email}
-                </a>
-              </Text>
-            </Paragraph>
-          </Card>
-          <Card title="General Ministry Contacts">
-            <Paragraph>
-              <Text strong className="ministry-contact-title">
-                Chief Inspector of Mines
-              </Text>
-              <br />
-              <Text>{Contacts.CHIEF_INSPECTOR.name}</Text>
-              <br />
-              <Text>{Contacts.CHIEF_INSPECTOR.phone}</Text>
-              <br />
-              <Text>
-                <a href={`mailto:${Contacts.CHIEF_INSPECTOR.email}`}>
-                  {Contacts.CHIEF_INSPECTOR.email}
-                </a>
-              </Text>
-            </Paragraph>
-            <Paragraph>
-              <Text strong className="ministry-contact-title">
-                Executive Lead (Authorizations)
-              </Text>
-              <br />
-              <Text>{Contacts.EXEC_LEAD_AUTH.name}</Text>
-              <br />
-              <Text>{Contacts.EXEC_LEAD_AUTH.phone}</Text>
-              <br />
-              <Text>
-                <a href={`mailto:${Contacts.EXEC_LEAD_AUTH.email}`}>
-                  {Contacts.EXEC_LEAD_AUTH.email}
-                </a>
-              </Text>
-            </Paragraph>
-          </Card>
+          <Row gutter={[0, 16]}>
+            <Col>
+              <Card title="Regional Ministry Contacts">
```

Comment:
To remove some duplicate code here, can you update the contacts to be an array of contacts and loop through them displaying in the `Paragraph` here, in the future the contact array will need to be the only thing updated and the view will automatically inherit the new contacts